app/templates/Bank_Detail_Banner.html.twig

```diff
@@ -1,131 +1,79 @@
-  }
-  .text-lightblue {
-    color: #9fccf3;
-   }
-  .digit-group {
-    margin-right: 5px;
-    letter-spacing: 1px;
-  }
-  .switch-button {
-    color: #479EE7;
-    font-size: 19px;
-
-    height: 42px;
-    padding: 0 15px;
-    margin-right: 15px;
-
-    border-radius: 5px;
-    border: 1px solid #9fccf3;
-
-    background: #ffffff;
-    background: -moz-linear-gradient(top, #ffffff 0%, #f7f7f7 100%);
-    background: -webkit-gradient(linear, left top, left bottom, color-stop(0%, #ffffff), color-stop(100%, #f7f7f7));
-    background: -webkit-linear-gradient(top, #ffffff 0%, #f7f7f7 100%);
-    background: -o-linear-gradient(top, #ffffff 0%, #f7f7f7 100%);
-    background: -ms-linear-gradient(top, #ffffff 0%, #f7f7f7 100%);
-    background: linear-gradient(to bottom, #ffffff 0%, #f7f7f7 100%);
-    filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#ffffff', endColorstr='#f7f7f7',GradientType=0 );
-  }
-  .banner-btn-close {
-    position: absolute;
-    top: 8px;
-    right: 8px;
-    cursor: pointer;
-  }
-</style>
 <script type="text/javascript">
-var timer, bankDetailBannerClosed = false;
-$( document ).ready( function() {
-  timer = window.setTimeout( function() {
-    $( '#bank-detail-banner' ).fadeIn();
-  }, 5000 );
-
-  $( 'input' ).focus( function() {
-    $( '#bank-detail-banner' ).fadeOut();
-    clearTimeout( timer );
-  } );
+	var timer, bankDetailBannerClosed = false;
+	$( document ).ready( function() {
+		timer = window.setTimeout( function() {
+			$( '.sliding-banner' ).fadeIn();
+		}, 5000 );
 
-  // This is for our fake radio buttons
-  $( 'label' ).click( function() {
-    $( '#bank-detail-banner' ).fadeOut();
-    clearTimeout( timer );
-  } );
+		$( 'input, label' ).focus( function() {
+			$( '.sliding-banner' ).fadeOut();
+			clearTimeout( timer );
+		} );
 
-  $( '.banner-btn-close, .slide-toggle' ).click( function() {
-    $( '#bank-detail-banner' ).fadeOut();
-    bankDetailBannerClosed = false;
-  } );
+		$( '.banner-btn-close, .slide-toggle' ).click( function() {
+			$( '.sliding-banner' ).fadeOut();
+			bankDetailBannerClosed = false;
+		} );
 
-  $( 'body' ).on( 'click',  '#wlightbox-close', function () {
-    if( !bankDetailBannerClosed ) {
-       $( '#bank-detail-banner' ).fadeIn();
-    }
-  } );
+		$( 'body' ).on( 'click',  '#wlightbox-close', function () {
+			if( !bankDetailBannerClosed ) {
+				$( '.sliding-banner' ).fadeIn();
+			}
+		} );
 
-  $( 'a.wlightbox' ).click( function() {
-    $( '#bank-detail-banner' ).fadeOut();
-    clearTimeout( timer );
-  } );
+		$( 'a.wlightbox' ).click( function() {
+			$( '.sliding-banner' ).fadeOut();
+			clearTimeout( timer );
+		} );
 
-  /* switch banner pages */
-  $( '.switch-button' ).click( function() {
-    $( '#bank-details' ).slideToggle();
-  } );
-} );
+		/* switch banner pages */
+		$( '.switch-button' ).click( function() {
+			var contentWrapper = $( '.content-wrapper' ),
+				newValue = contentWrapper.css( 'top' ) === '0px'
```

Comment:
`jQuery.css()`
"When a number is passed as the value, jQuery will convert it to a string and add px to the end of that string. _If the property requires units other than px, convert the value to a string and add the appropriate units before calling the method._" (emphasis mine) TIL